qwik-graphql-tailwind/src/components/user-dropdown/user-dropdown.tsx

```diff
@@ -24,8 +25,17 @@ export const UserDropdown = component$(({ image, username }: UserDropdownProps)
     store.expanded = false;
   });
 
-  const signOut$ = $(() => {
-    // TODO: sign out
+  const signOut$ = $(async () => {
+    sessionStorage.removeItem(AUTH_TOKEN);
+    fetch(SIGN_OUT_URL, {
```

Comment:
why do we have to do this please?

Reply:
This is a feature of Qwik, [here](https://qwik.builder.io/docs/advanced/optimizer/#the-meaning-of-)

Reply:
Can you elaborate on this? what exactly is the issue